xml/smtp_banners.xml

```diff
@@ -1636,7 +1636,16 @@ The system or service fingerprint with the highest certainty overwrites the othe
     <param pos="0" name="service.product" value="Perl"/>
     <param pos="1" name="host.name"/>
   </fingerprint>
-  <fingerprint pattern="^([^ ]+) E?SMTP(?: (?:Service )?Ready\.?)?$" flags="REG_ICASE">
+<fingerprint pattern="Cellopoint E-mail Firewall (v\d+.\d+.\d+) Build (\d+) ready">
+	<description>Matches a Cellopoint Firewall catching the request</description>
+	<example> Cellopoint E-mail Firewall v3.9.11 Build 0730 ready </example>
+	<param pos="0" name="service.vendor" value="Cellopoint"/>
+	<param pos="0" name="service.family" value="Email Firewall"/>
+	<param pos="0" name="service.product" value="Email Firewall"/>
+	<param pos="1" name="service.version"/>
+	<param pos="2" name="service.version.version"/>
```

Comment:
I'm not 100% sure this should be here.  Do you have other examples of the build values, in particular as they related to the `service.version`?  For example, do you know if there will ever be other build versions for 3.9.11?